lib/services/devices/deviceRegistryMongoDB.js

```diff
@@ -188,26 +188,11 @@ function listDevices(type, service, subservice, limit, offset, callback) {
     });
```

Comment:
PR should include an entry in CHANGES_NEXT_RELEASE file describing the change.

Reply:
Fixed 39c7874